### PR TITLE
Rework organization controller

### PR DESCRIPTION
**Pull request by vasya539**
- [x] merged with the latest master branch
- [x]resolved all the conflicts, PR could be merged
- [x] new code has new specs
- [o] for new models - new factories were added too
- [x] unit specs are passed
- [x] code is well-formatted, Ruby, CSS, etc
- [x] screenshots of new pages, features are added
- [x] code passed internal team code review

PS: delete your branch in github after merge http://prntscr.com/q80mvq

### CODE REVIEW

**Comment by vasya539** [comment]
image

**Comment by vasya539** [comment]
image

**Comment by vasya539** [comment]
image

File: app/views/account/my_organization/edit.html.slim [hunk 1]
```diff
@@ -0,0 +1,19 @@
+- breadcrumbs [\
+  { home: true, url: account_path },\
+    \
```
Comment (vikdotdev) [inline app/views/account/my_organization/edit.html.slim]: ці бекслеші лишні

File: app/views/account/my_organization/show.html.slim [hunk 2]
```diff
@@ -0,0 +1,33 @@
+- breadcrumbs [\
+  { home: true, \
+    url: account_path },\
+    \
```
Comment (vikdotdev) [inline app/views/account/my_organization/show.html.slim]: те саме

**Comment by vasya539** [comment]
image

File: app/controllers/account/my_organization_controller.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
+class Account::MyOrganizationController < Account::AccountController
+  before_action :check_if_admin?
```
Comment (igorkasyanchuk) [inline app/controllers/account/my_organization_controller.rb]: тут точно все правильно?

File: spec/controllers/account/my_organization_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+      it 'updates organization name' do
+        patch :update, params: {
+          organization: {
+            name: 'NewName'
+          },
+          id: organization.id
+        }
+
+        expect(response).to redirect_to(account_my_organization_path)
+        expect(assigns(:organization).name).not_to eq('OldName')
```
Comment (igorkasyanchuk) [inline spec/controllers/account/my_organization_controller_spec.rb]: мені не подобається що ти перевіряєш змінну якусь 
а не те що в базі

зроби так

organization.reload

expect(organization.name).not_to eq('OldName')
... i t.d.

**Pull request merged**